libraries/botbuilder-runtime-integration-restify/src/index.ts

```diff
@@ -30,32 +38,36 @@ const defaultOptions: Options = {
 export async function start(
     applicationRoot: string,
     settingsDirectory: string,
-    options = defaultOptions
+    options: Partial<Options> = {}
 ): Promise<void> {
+    const { port, messagingEndpointPath } = TypedOptions.check(Object.assign({}, defaultOptions, options));
     const [services, configuration] = await getRuntimeServices(applicationRoot, settingsDirectory);
-    const server = await makeServer(services, configuration);
 
-    server.listen(options.port, () => {
-        console.log(`server listening on port ${options.port}`);
+    const server = await makeServer(messagingEndpointPath, services, configuration);
+
+    server.listen(port, () => {
+        console.log(`server listening on port ${port}`);
     });
 }
 
 /**
  * Create a server using the runtime restify integration.
  *
+ * @param messagingEndpointPath path for receiving and processing [Activities](xref:botframework-schema.Activity)
  * @param services runtime service collection
  * @param configuration runtime configuration
  * @returns a restify server ready to listen for connections
  */
 export async function makeServer(
```

Comment:
This function should accept `Partial<Options>` as well and support the same default options resolution - that makes the developer experience consistent even if a user wants to customize the restify server.